### DISCUSSION TITLE
Wallet will not sync

### DISCUSSION THREAD

**Discussion by glukkon24** [Chia-Network/chia-blockchain#7320] [Support]
I cannot get wallet to sync. I have deleted wallet twice and tried to resync and now sitting at 512,566 with it toggling between syncing and not synced. I have tried everything.

**Comment by DaveKoz** [comment]
I have this issue in windows, on Mac OS 11.3.1 it is working fine

**Reply by glukkon24** [reply]
have you figured anything out I am still trying to sync for the 3rd time

**Reply by DaveKoz** [reply]
No i have not. I'm going to try a brand new  install on windows 10 pro. I'll let you know. Make sure you have the 24 word key to reinstall wallet if you try it.

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
